stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -87,6 +87,10 @@ spec:
             - containerPort: {{ .Values.Master.JMXPort }}
               name: jmx
             {{- end }}
+            {{- if .Values.Master.BuildInfoProxyPort }}
```

Comment:
Can we make this generic? Something like:

```
Master:
  # AddtionalPorts:
  # - Name: arm-proxy
  #   ContainerPort:   9099
```

I'm trying to avoid having to do this for each plugin that needs something like it.

Reply:
Sounds good. Will try to include it in next commit.